modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -832,6 +832,14 @@ return view.extend({
 						}
 					};
 
+					so = ss.taboption('ipv6-ra', form.Value, 'ra_pref64', _('NAT64 prefix'), _('Announce NAT64 prefix in <abbr title="Router Advertisement">RA</abbr> messages.'));
```

Comment:
@jow considering these are translated string is it ok to put html stuff here?

Reply:
Okay but not ideal. Embedded HTML tends to get broken by translators. It also requires extra care to be XHTML compliance.